bobtemplates/plone/content_type/profiles/default/types/+dexterity_type_fti_file_name+.xml.bob

```diff
@@ -51,21 +51,18 @@
 
   <!-- Enabled behaviors -->
   <property name="behaviors" purge="false">
-{{% if not dexterity_type_create_class %}}
-    <element value="{{{ package.dottedname }}}.content.{{{ dexterity_type_name_normalized }}}.I{{{ dexterity_type_name_klass }}}"/>
-{{%  endif %}}
```

Comment:
@MrTango This does not make any sense to me, so I remove it here. The schema is never registered as a behavior nor does it make sense to do so. 
If you or anyone else have any idea why someone tried to do this, please explain.

Reply:
btw, the plain-schema-as-a-behavior is deprecated in plone.dexterity and marked for removal with the next major of plone.dexterity